Web Site Pages/SRVvsIRV.txt

```diff
@@ -15,25 +17,47 @@ Traditional Instant Runoff Voting (IRV) is the Ranked Choice O.G. - it's been us
+<h4>SRV uses runoff between two top scoring candidates</h4>
+
+<p>In SRV, the final contestants are the two overall highest scoring candidates. The winner is the one with more ballots that show a preference for that candidate, based on those voters who gave a different score to each of the two.
+</p>
 
 <div class="gray-box">
 <br>
 <strong>Honesty:</strong> Can the voter safely express her honest opinion on the ballot, and likewise, to what level does the system disincentivize voters from strategically voting insincerely in order to produce a better outcome?
 <br><br>
 </div>
 
-<p>Instant Runoff Voting proponents <a href="http://www.betterballotbenton.com/the-benefits">often make the inaccurate claim</a> that with IRV, "you can honestly rank candidates in order of choice without having to worry about how others will vote and who is more or less likely to win." In fact, you can only safely rank candidates honestly in IRV if your favorite either has no chance at all or is a very strong candidate. This video shows clearly how IRV suffers from a more general form of the vote-splitting spoiler effect:</p>
+<h4>With IRV, voters can't always vote safely for honest favorite as 1st choice</h4>
+
+<p>Instant Runoff Voting proponents <a href="http://www.betterballotbenton.com/the-benefits">often make the inaccurate claim</a> that with IRV, "you can honestly rank candidates in order of choice without having to worry about how others will vote and who is more or less likely to win." In fact, you can only safely rank candidates honestly in IRV if your favorite either has no chance at all or is a very strong candidate. Giving your top vote to your favorite in IRV can actually help elect your least favorite candidate. This video shows clearly how IRV suffers from a more general form of the vote-splitting spoiler effect:</p>
 
 https://www.youtube.com/watch?v=JtKAScORevQ
 
-<p>IRV advocates argue that <a href="http://www.fairvote.org/new_lessons_from_problems_with_approval_voting_in_practice">rating systems are vulnerable to tactical 'bullet voting'</a> - that is, support of just one candidate on the ballot in order to maximize the chance of one's favorite choice winning. Rating advocates have demonstrated that in a significant number of IRV elections, giving full support to your favorite can actually help elect your least favorite candidate.
+<h4>Plain score voting has potential strategic issues</h4>
+<p>IRV advocates argue that <a href="http://www.fairvote.org/new_lessons_from_problems_with_approval_voting_in_practice">rating systems are vulnerable to tactical 'bullet voting'</a> — that is, support of just one candidate on the ballot in order to maximize the chance of one's favorite choice winning. Another strategic degrading that could happen with score is maximizing and minimizing the scores of all the candidates in order to play it safe. That means score voting would become simply approval voting.
```

Comment:
I still like emphasizing that extremed scores just equals approval, i.e. it doesn't degrade down to plurality or worse, just not not-ideal. So I like my added sentence here.